kratos/solving_strategies/schemes/residual_based_bdf_displacement_scheme.h

```diff
@@ -111,9 +113,9 @@ class ResidualBasedBDFDisplacementScheme
     /**
      * Clone
      */
-    BaseTypePointer Clone() override
+    typename BaseType::Pointer Clone() override
     {
-        return BaseTypePointer( new ResidualBasedBDFDisplacementScheme(*this) );
+        return Kratos::make_shared<ResidualBasedBDFDisplacementScheme>(*this);
```

Comment:
this is not nice. my impression is that we should delete the copy constructor and implement this a bit more carefully

Reply:
What do I do?